src/Evolution/DiscontinuousGalerkin/MortarInterpolator.hpp

```diff
@@ -0,0 +1,109 @@
+// Distributed under the MIT License.
```

Comment:
Is this somehow tied to evolutions? Otherwise please put this into a more general place, e.g. `NumericalAlgorithms/Interpolation/` or `NumericalAlgorithms/DiscontinuousGalerkin/`.